main.js

```diff
@@ -437,7 +444,7 @@ function getRowOfLastVisibleCell() {
     const numberOfColumns = columnInput.value;
     for(i = items.length - 1; i >= 0; i--) {
         //New index in order to generate JSON with the new order of elements reflacted
-        let itemID = items[i].getAttribute('id');
+        let itemID = items[i].getAttribute("id").substr(-1); 
```

Comment:
JS generally does single quotes. A linter would have fixed this if this project had one, so I'm assuming it doesn't? Either way, could you fix this one and the one on line 474?

Reply:
👍